pages/dmp_supporting_info/support_08_preserve_publish.md

```diff
@@ -63,67 +63,72 @@ When deciding where to archive your data, make suere you chose an archive where
-
-Generally, institutions and research funders set requirements to retain data for a given period. For example, the Research Council of Norway "has stipulated as a requirement that research data must be stored in a safe and secure manner for a minimum of 10 years" [(Policy for Open Science, 2020)](https://www.forskningsradet.no/siteassets/forskningspolitisk-radgivning/apen-forskning/nfr-policy-open-science-eng.pdf). On the other hand, the Research Council of Norway states that research projects "should explicitly address how research data that are not considered to have long-term value should be managed, or if relevant destroyed, after a certain period of time."[(Policy for for Open Access to Research
-Data, 2017)](https://www.forskningsradet.no/siteassets/forskningspolitisk-radgivning/apen-forskning/nfr-policy-open-science-eng.pdf)
+Determining which data are of value and should be preserved or whether data need to or can be deleted, is a decision that should be taken actively. If data(sets) are or will be deleted, they can be listed in this question, including a brief description of the reasons for deletion. There may be legal (e.g. defined by the Health Research Act) or contractual obligations to delete data.
 
-Yet, it can be worthwhile to consider which data in a project are not of value (e.g. data from failed experiments), only have intermediate value (e.g. during data analysis) and which data should be preserved long-term, particularly if storage of large amounts of data is associated with significant costs.
+Generally, institutions and research funders set requirements to retain data for a given period. For example, the Research Council of Norway "has stipulated as a requirement that research data must be stored in a safe and secure manner for a minimum of 10 years" [(Policy for Open Science, 2020)](https://www.forskningsradet.no/siteassets/forskningspolitisk-radgivning/apen-forskning/nfr-policy-open-science-eng.pdf). On the other hand, the Research Council of Norway states that research projects "should explicitly address how research data that are not considered to have long-term value should be managed, or if relevant destroyed, after a certain period of time." [(Policy for for Open Access to Research
+Data, 2017)](https://www.forskningsradet.no/contentassets/6ccb7b14c18a48ee974acd78fc64efba/the-research-council-of-norways-policy-for-open-access-to-research-data.pdf)
 
+So, it can be worthwhile to consider which data in a project are not of value (e.g. data from failed experiments), only have intermediate value (e.g. during data analysis) and which data should be preserved long-term, particularly if storage of large amounts of data is associated with significant costs. But also, do take into consideration that data not considered to be of any value for your present need, may turn out to be of value to others, or even to yourself, some time into the future.
```

Comment:
```suggestion
So, it can be worthwhile to consider which data in a project are not of value (e.g. data from failed experiments), only have intermediate value (e.g. during data analysis) and which data should be preserved long-term. This is particularly important if storage of large amounts of data is associated with significant costs. However, take into consideration that data not considered to be of any value for your present need, may turn out to be of value to others, or even to yourself, some time into the future.
```